articles/flutter-vs-kotlin/index.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/flutter-vs-kotlin/
+title: Flutter vs. Kotlin
+description: This article goes over a comparison of Flutter and Kotlin - talking the advantages and disadvantages of both.
+author: Bavneet Singh
+date: 2020-09-01T00:00:00-12:00
+topics: [Node.js]
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/flutter-vs-kotlin/hero.jpg
+    alt: Kotlin vs Flutter image example
+---
+<!--more-->
+Mobile application development has become an essential part of the software business in recent years. Developers used to prefer ReactNative (or Flutter to a lesser extent) to develop cross-platform applications due to their low-cost features. Today, another cross-platform language named Kotlin has also managed to enter into the world of cross-platform development. Since then Flutter has also managed to improve a lot in the past years, and it now provides better features than ReactNative.  
```

Comment:
This section is not informative enough. A lot of background missing:

- Why did/do developers prefer React Native?
- What distinguishes React Native from Flutter?
- When did Kotlin emerge? Who created it? When was Flutter created?
- Overview of the differences between the two
- What exactly has improved in past years, and what are its better features?